src/pages/Chat/index.tsx

```diff
@@ -16,7 +18,7 @@ const ChatPage: Component = () => (
       class="flex h-full flex-col-reverse"
     >
       <InputBar />
-      <div class="flex flex-col-reverse overflow-y-scroll">
+      <div class="chatbox flex flex-col-reverse overflow-y-scroll">
```

Comment:
What does this do? I couldn't find any references to `chatbox` within the code.

Reply:
This is a remnant of testing, will remove this.